README.md

```diff
@@ -188,24 +164,45 @@ func fetchExistingPaymentMethod(clientToken: String) {
 
 ### Localization
 
-Drop-In is currently localized for [23 languages](https://github.com/braintree/braintree-ios-drop-in/tree/master/BraintreeUIKit/Localization).
+Drop-In is currently localized for [22 languages](https://github.com/braintree/braintree-ios-drop-in/tree/master/BraintreeUIKit/Localization).
 
 ### Color Schemes
 
 Drop-In is fully customizable, but we also provide `Light`, `Dark` and `Dynamic` color schemes. The dynamic color scheme will switch between light and dark based on whether the device is in light or dark mode. The `Dynamic` scheme is only available in iOS 13 or higher. Drop-In will use the `Light` color scheme by default.
 ```swift
-// Set the theme before initializing Drop-In
-BTUIKAppearance.sharedInstance().colorScheme = .dynamic
+let uiCustomization = BTDropInUICustomization(colorScheme: .dark)
+
+let dropInRequest = BTDropInRequest()
+dropInRequest.uiCustomization = uiCustomization
 ```
 
 ![Drop-in dark theme](Images/client-sdk-ios-series-dark.png "Drop-in dark theme")
 
 ### Customization
 
-Use `BTUIKAppearance` to customize the appearance of Drop-In and other BraintreeUIKit classes.
+Use `BTDropInUICustomization` to customize the appearance of Drop-In.
+
+```swift
+let uiCustomization = BTDropInUICustomization(colorScheme: .dynamic)
+uiCustomization.fontFamily = "Helvetica"
+uiCustomization.boldFontFamily = "Helvetica Bold"
+
+let dropInRequest = BTDropInRequest()
+dropInRequest.uiCustomization = uiCustomization
+```
+
+### Accessibility
+
+#### Dynamic Type
+
+Dynamic type is enabled by default for both system and custom fonts. We strongly encourage you to leave this feature on, but if you need to disable it, you may do so by setting `disableDynamicType` to `true` on `BTDropInUICustomization`.
```

Comment:
```suggestion
Dynamic type is enabled by default for both system and custom fonts. We strongly encourage you to leave this feature on, but if you wish to disable it, you may do so by setting `disableDynamicType` to `true` on `BTDropInUICustomization`.
```
take-or-leave: I feel like softer language here could further discourage merchants from doing this lol.